README.md

```diff
@@ -40,10 +40,19 @@ async function handleEvent(event) {
     try {
       return await getAssetFromKV(event, { mapRequestToAsset: customKeyModifier })
     } catch (e) {
-      return new Response(`"${customKeyModifier(event.request.url)}" not found`, {
-        status: 404,
-        statusText: 'not found',
-      })
+      if (resp instanceof KVError) {
+        switch (e.code) {
+          case 404:
+            return notFoundResponse
```

Comment:
```suggestion
            return new Response(`${event.request.url} was not found`
```

Reply:
Shouldn't this pull from 404.html in KV?

Reply:
This is just the README so i wanted to keep it as simple as possible